terraform/cloudfoundry/router_elb.tf

```diff
@@ -8,6 +8,7 @@ resource "aws_elb" "cf_router" {
     "${aws_security_group.pingdom-probes-0.id}",
     "${aws_security_group.pingdom-probes-1.id}",
     "${aws_security_group.pingdom-probes-2.id}"
```

Comment:
Sorry dude, travis demands an additional comma and I don't have the authority to overrule it.